### PR TITLE
Feature/aut user information

### PR DESCRIPTION
**Pull request by serqiiiio**
Send to develop the cypress project

### CODE REVIEW

File: cypress/integration/common/verifysteps.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Then } from "cypress-cucumber-preprocessor/steps";
+
+Then("I should see the information updated in the table", (dataTable) => {
```
Comment (ATCarlosG) [inline cypress/integration/common/verifysteps.js:3]: Have you executed your two tests? from what Im looking, this step will compare:
firstName Paulo (from datatable), against Juan (from fixture) making it fail

File: cypress/integration/createUser.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@Create
```
Comment (ATCarlosG) [inline cypress/integration/createUser.feature:1]: Put an ID on your scenarios so it makes easy to identify them, also the type of the test (functional, acceptance, negative, boundary... etc)

**Review by ATCarlosG — CHANGES_REQUESTED** [review]
Review the entire execution of your tests, I think there might be a false negative
